portal-api/api-tests/v1/gef/amendments/amendments.patch.api-test.ts

```diff
@@ -160,6 +160,7 @@ describe('/v1/gef/facilities/:facilityId/amendments/:amendmentId', () => {
           createdAt: 1702061978881,
           updatedAt: 1702061978881,
           status: PORTAL_AMENDMENT_STATUS.DRAFT,
+          eligibilityCriteria: { version: 1, criteria: [] },
```

Comment:
I think we should declare a const in `libs/common` for latest EC version and pick 1 from there, I would expect it look like below

```ts
const AMENDMENT = {
ELIGIBLITY_CRITERIA: {
VERSION: {
LATEST: 1,
ONE: 1,
TWO: 2,
}
}
};
```

Reply:
As discussed offline will leave this for now, as I don't think the version numbers will be used in the code much